premake5.lua

```diff
@@ -302,6 +302,16 @@ function addSourceDir(path)
         path .. "/*.hpp",       -- C++ style headers (for glslang)
         path .. "/*.natvis",    -- Visual Studio debugger visualization files
     }
+    removefiles
```

Comment:
Can't we just use "**/*generated.h",
        "**/*generated-macro.h"
To cover ast and ref-object scenarios?